cmd/goal/accountsList.go

```diff
@@ -225,12 +226,30 @@ func (accountList *AccountsList) outputAccount(addr string, acctInfo v1.Account,
 		fmt.Printf("\t[%d/%d multisig]", multisigInfo.Threshold, len(multisigInfo.PKs))
 	}
 	if len(acctInfo.AssetParams) > 0 {
-		fmt.Printf("\t[created assets:")
+		fmt.Printf("\t[created asset IDs: ")
+		var out []string
 		for curid, params := range acctInfo.AssetParams {
-			fmt.Printf(" %d (%d %s)", curid, params.Total, params.UnitName)
+			out = append(out, fmt.Sprintf("%d (%d %s)", curid, params.Total, params.UnitName))
 		}
-		fmt.Printf("]")
+		fmt.Printf("%s]", strings.Join(out, ", "))
 	}
+	if len(acctInfo.AppParams) > 0 {
+		fmt.Printf("\t[created app IDs: ")
```

Comment:
nit: unify the 
```
fmt.Printf("\t[created app IDs: ")
...
fmt.Printf("%s]", strings.Join(out, ", "))
```

into 
```
fmt.Printf("\t[created app IDs: %s]", strings.Join(out, ", "))
```

Reply:
opted-in assets are not printed here -- they are printed elsewhere along with frozen status, creator, etc.

Reply:
done!